### DISCUSSION TITLE
建议dialog新增动画控制

### DISCUSSION THREAD

**Discussion by yudan215** [element-plus/element-plus#1344] [Ideas]
1.默认最好能实现类似https://ant.design/components/modal-cn/ 的动画方式
根据按钮位置打开和隐藏的动画效果

2.可以配置根据页面大小响应式（非样式覆盖）

3.是否允许拖拽

4.loading状态

**Comment by jw-foss** [comment]
Quick answer:
1. The animation is our `Design Language`, it is the backbone of our project, a UI library is all about the design language, we cannot change that.
2. ElementPlus is a desktop UI library, we provide limited responsive feature for our user incase they use it on the desktop with a small break point but in the end it is still a desktop UI library.
3. This one is negotiable, we can implement that for dialog like ant design did, but still we need to discuss with out designers.
4. Loading is controlled by users like you, you can customize the footer with button to add a `loading` state to it. That's all up to you. Hope I have answered your questions clearly, if you have any further question feel free to ask.
